YouVsVirus/Assets/Scenes/leveldisco/EndLevelControllerLeveldisco.cs

```diff
@@ -16,37 +16,26 @@ public class EndLevelControllerLeveldisco : EndLevelControllerBase
     /// </summary>
     public void NotifyFriendExposed()
     {
-        if (!CanvasFail.activeInHierarchy)
+        if (!levelHasFinished && !CanvasFail.activeInHierarchy)
         {
             CanvasFail.SetActive(true);
             failText.text = "Sorry.\n\nYour friend is infected.";
+            FindAndPlaceHuman(CanvasFail);
+            levelHasFinished = true;
         }
     }
     /// <summary>
     /// Failure due to healthy player exit without friend
     /// </summary>
     public void PlayerExitHealthyWithoutFriend()
     {
-        CanvasFail.SetActive(true);
-        failText.text = "You got home safely but Sandra is still at the party.";
-    }
-
-    /// <summary>
-    /// Failure due to sick friend or player exit together
-    /// </summary>
-    public void PlayerOrFriendExitSick()
-    {
-        CanvasFail.SetActive(true);
-        failText.text = "Sorry.\nYou got out but were exposed to the virus.";
-    }
-
-    /// <summary>
-    /// Failure due to sick player exit withou friend
-    /// </summary>
-    public void PlayerExitSickWithoutFriend()
-    {
-        CanvasFail.SetActive(true);
-        failText.text = "You were exposed to the virus and Sandra is still at the party.";
```

Comment:
Why are they gone? Are they elsewhere? They still occur in the game so i assume you moved the code...?

Reply:
They cannot occur anymore. Since the game ends when the player or the friend is infected both cannot move any longer. Hence player and friend cannot exit sick and the player can't exit sick.